randommenu_platinum.rb

```diff
@@ -0,0 +1,66 @@
+# Random Menu Generator
+
+# create empty arrays
+adjective = []
+cook_method = []
+food_item = []
+
+# Get user input for how many item menus
+puts "Hey! Let's generate a random menu for you.\nHow many items would you like on your menu?"
+list_num = gets.chomp
+
+# Error message for invalid number
+until list_num.to_i.to_s == list_num && list_num.to_i > 0
```

Comment:
Nice quality control here. Converting `list_num` multiple times here is a little hard to follow, though. If you're trying to assert that the number your user has entered is an actual number and it's greater than zero, you should be able to get away with just writing:
```ruby
until list_num.to_i > 0
```
(Strings that can't convert directly to numbers are converted to zero and you're making sure their input is greater than zero already.)